README.md

```diff
@@ -154,23 +214,54 @@ The final bot provided is a QueryBot.
 This bot lets you query a collection of documents.
 To use it, you have two options:
 
-1. Pass in a list of paths to text files, or
-2. Pass in a session name of a previously instantiated `QueryBot` that model. (This will load the previously-computed text index into memory.)
+1. Pass in a list of paths to text files and make Llamabot create a new collection for them, or
+2. Pass in the `collection_name` of a previously instantiated `QueryBot` model. (This will load the previously-computed text index into memory.)
 
-As an illustrative example:
+An illustrative example creating a new collection:
 
 ```python
 from llamabot import QueryBot
 from pathlib import Path
 
-blog_index = Path("/path/to/index.json")
-bot = QueryBot(system_message="You are an expert on Eric Ma's blog.", session_name="eric_ma_blog")  # this loads my previously-embedded blog text.
-# alternatively:
-# bot = QueryBot(system_message="You are an expert on Eric Ma's blog.", session_name="eric_ma_blog", document_paths=[Path("/path/to/blog/post1.txt"), Path("/path/to/blog/post2.txt"), ...])
+bot = QueryBot(
+  system_prompt="You are an expert on Eric Ma's blog.", collection_name="eric_ma_blog",
+  document_paths=[
+    Path("/path/to/blog/post1.txt"),
+    Path("/path/to/blog/post2.txt"), ...
+  ],
+  # Optional:
+  # model_name="gpt-3.5-turbo"
+  # or
+  # model_name="ollama/mistral"
+) # This creates a new embedding for my blog text.
 result = bot("Do you have any advice for me on career development?")
 display(Markdown(result.response))
 ```
 
+An illustrative example using an already existing collection:
+
+```python
+from llamabot import QueryBot
+
+bot = QueryBot(
+  system_prompt="You are an expert on Eric Ma's blog",
+  collection_name="eric_ma_blog",
+  # Optional:
+  # model_name="gpt-3.5-turbo"
+  # or
+  # model_name="ollama/mistral"
+)  # This loads my previously-embedded blog text.
+result = bot("Do you have any advice for me on career development?")
+display(Markdown(result.response))
+```
+
+For more explanation about the `model_name`, see [the examples with `SimpleBot`](#using-simplebot-with-a-local-ollama-model).
+
+#### `QueryBot` Usage Tips
```

Comment:
I'd love to have this section moved to a different place. It can be a much more fleshed out Jupyter notebook that outlines "How to develop a QueryBot", perhaps? In this way, it can, tutorial-style, show how to build RAG applications by first considering specific use cases before we go to more general ones.

Reply:
OK, good point. :+1: 
For now:

1. I removed this section from the `README`.

2.
I created an issue for updating the `QueryBot` tutorial and added a comment in that issue:
https://github.com/ericmjl/llamabot/issues/66#issuecomment-2227113131

Feel free to remove the section there and add it to a more suitable place.

Reply:
Thanks, that's a great move!